shell/platform/common/cpp/public/flutter_texture_registrar.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_COMMON_CPP_PUBLIC_FLUTTER_TEXTURE_REGISTRAR_H_
+#define FLUTTER_SHELL_PLATFORM_COMMON_CPP_PUBLIC_FLUTTER_TEXTURE_REGISTRAR_H_
+
+#include <stddef.h>
+#include <stdint.h>
+#include <memory>
+
+#include "flutter_export.h"
+
+#if defined(__cplusplus)
+extern "C" {
+#endif
+
+// Opaque reference to a texture registrar.
+typedef struct FlutterDesktopTextureRegistrar*
+    FlutterDesktopTextureRegistrarRef;
+
+struct GLFWPixelBuffer {
```

Comment:
Please don't use `GLFW` as a type prefix; it makes in confusing for the reader, since this is not actually a GLFW type.